api/proto/teleport/legacy/types/types.proto

```diff
@@ -1276,25 +1276,24 @@ message ProvisionTokenSpecV2 {
 // ProvisionTokenSpecV2
 message ProvisionTokenSpecV2TPM {
   message Rule {
+    reserved 2, 3;
```

Comment:
Linters will want you to reserve/change the names too.

Reply:
Hmm - I don't really want to change the name.

Reply:
I've got bad news for you...

Reply:
:(

Reply:
I'll come up with a better but slightly similar name 😂

Reply:
Brilliant  😂